services/drupal/web/themes/epa_theme/source/_patterns/00-config/01-mixins/_mixins.grids.scss

```diff
@@ -38,8 +39,8 @@
 ) {
   display: flex;
   flex-wrap: wrap;
-  margin-left: rem(-$gutter-width / 2);
-  margin-right: rem(-$gutter-width / 2);
+  margin-left: rem(math.div($gutter-width, 2));
+  margin-right: rem(math.div($gutter-width, 2));
```

Comment:
@liquilife update `$gutter-width` to be negative  -> `-$gutter-width`.
See lines 45/46 of https://github.com/forumone/gesso/blob/5.x/source/00-config/mixins/_grids.scss

Reply:
@tomealter I just updated this. `$gutter-width` is now set to negative where it was previously also negative`-$gutter-width`.